Precise the the behaviour of the reduce (#22796)
mdn/content: files/en-us/web/javascript/reference/global_objects/array/reduce/index.md

- `initialValue` {{optional_inline}}
  - : A value to which `accumulator` is initialized the first time the callback is called.

>     If `initialValue` is specified, that also causes `currentValue` to be initialized to the first value in the array.

Reviewer: ```suggestion
    If `initialValue` is specified, `callbackFn` starts executing with the first value in the array as `currentValue`.
```

>     If `initialValue` is _not_ specified, `accumulator` is initialized to the first value in the array, and `currentValue` is initialized to the second value in the array. If `reduce()` is called on an empty array without providing an `initialValue`, an error will occur.

Reviewer: ```suggestion
    If `initialValue` is _not_ specified, `accumulator` is initialized to the first value in the array, and `currentValue` is initialized to the second value in the array. In this case, if the array is empty (so that there's no first value to return as `accumulator`), an error is thrown.
```
